etl/steps/data/garden/countries/2023-09-29/cow_ssm.py

```diff
@@ -1,10 +1,18 @@
 """Load a meadow dataset and create a garden dataset."""
 
+import owid.catalog.processing as pr
+import pandas as pd
+from owid.catalog import Table
+
 from etl.data_helpers import geo
 from etl.helpers import PathFinder, create_dataset
 
 # Get paths and naming conventions for current step.
 paths = PathFinder(__file__)
+# Only for table tb_regions:
+# The current list of members goes until 2016, we artificially extend it until 2023, preserving the last value
+EXPECTED_LAST_YEAR = 2016
+LAST_YEAR = 2022  # Update to extend it further in time
```

Comment:
Do you want to extend to 2023? Because it seems it extends to 2022

Reply:
I actually mean 2022, because 31st December 2023 hasn't yet occurred. I'll update the comment.